test/e2e_node/conformance_test.go

```diff
@@ -62,7 +62,7 @@ var _ = Describe("Container Conformance Test", func() {
 				phase := api.PodPending
 				for start := time.Now(); time.Since(start) < serviceCreateTimeout; time.Sleep(time.Second * 30) {
 					ccontainer, err := terminateCase.Get()
-					if err != nil || ccontainer.Phase != api.PodPending {
+					if err != nil && ccontainer.Phase != api.PodPending {
```

Comment:
Looking at https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/container.go#L93, won't the phase always be `""` when `err != nil`? It seems like maybe you just want to wait until `ccontainer.Phase != api.PodPending`?